testsuite/tests/kuadrant/authorino/operator/tls/mtls/test_mtls_identity.py

```diff
@@ -23,10 +23,19 @@ def test_mtls_success(envoy_authority, valid_cert, envoy):
         assert response.status_code == 200
 
 
-@pytest.mark.parametrize("cert_authority, certificate, err, err_match", [
-    pytest.param("envoy_authority", "invalid_cert", ReadError, "unknown ca", id="Invalid certificate"),
-    pytest.param("invalid_authority", "valid_cert", ConnectError, "certificate verify failed", id="Unknown authority"),
-])
+@pytest.mark.parametrize(
+    "cert_authority, certificate, err, err_match",
+    [
+        pytest.param("envoy_authority", "invalid_cert", ReadError, "unknown ca", id="Invalid certificate"),
+        pytest.param(
+            "invalid_authority",
+            "valid_cert",
+            ConnectError,
+            "certificate verify failed",
+            id="Unknown authority",
+        ),
```

Comment:
I dislike the inconsistency here.

Reply:
it is consistent, the length is either below max line length or above it. It's everywhere same